lib/active_record/safer_migrations/postgresql_adapter.rb

```diff
@@ -3,23 +3,25 @@
 module ActiveRecord
   module SaferMigrations
     module PostgreSQLAdapter
-      SET_SETTING_SQL = <<-SQL.freeze
+      SET_SETTING_SQL = <<-SQL.
       UPDATE
         pg_settings
       SET
         setting = :value
       WHERE
         name = :setting_name
       SQL
+        freeze
```

Comment:
This is really messed up.

Reply:
Rubby, ladies and gentlemen

Reply:
![rubby_way](https://user-images.githubusercontent.com/237226/37965882-8d20cd0a-31be-11e8-8e92-e95c54726fa7.png)